### DISCUSSION TITLE
Como integro com um banco de dados?

### DISCUSSION THREAD

**Discussion by develogo** [frontendbr/forum#703] [Discussão]
Então pessoal, achei um repositorio (https://github.com/redhoodie/pizza) bastante interessante e to tentando aprender como eu crio um DB para isso mas eu não consigo. Por favor, alguem poderia me indicar como eu posso fazer, só apontar o caminho e eu irei estudar. Muito obrigado

**Comment by TiagoDanin** [comment]
Tenho duas soluções:
- 1. Tem uma biblioteca para javascript [jssql](https://www.npmjs.com/package/jssql) que é bem simples de usar. Mas porem tem que ter cuidado ao implementa para não exibir a senha do SQL para o usuário. 

- 2. Use o [Flask](https://pypi.python.org/pypi/Flask) para criar um web-server com o [Pizza](https://github.com/redhoodie/pizza), e crie um ponto para recebe as requisições de formulário e passa para um banco de dados SQL usando [dataset](https://dataset.readthedocs.io/en/latest/)

**Comment by doug2k1** [comment]
Se você quer focar em construir o front, fazer as chamadas e tratar os retornos, etc. eu recomendo algo simples como o https://github.com/typicode/json-server, que cria uma API REST pra você em dois palitos.
